python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -214,7 +214,7 @@ def isType(_obj, type_or_seq):
     if isinstance(type_or_seq, getKnownLists()):
         clean_list = []
         for type_obj in type_or_seq:
-            str_type = type('')
+            str_type = str
```

Comment:
There's no need for this temp variable. Just replace `str_type` with `str` below.